geppetto.js/geppetto-ui/src/3d-canvas/threeDEngine/TrackballControls.js

```diff
@@ -143,9 +143,6 @@ THREE.TrackballControls = function (object, domElement, handler) {
   };
 
   this.setCameraByConsole = function () {
-    if (_this.cameraByConsoleLock) {
```

Comment:
What's the reason behind this change? In #175 I ported TrackballControls to the latest version so whatever we change here now, needs to be ported again in #175